modules/apps/site-initializer/site-initializer-testray/site-initializer-testray-test/src/testFunctional/macros/TestrayCases.macro

```diff
@@ -0,0 +1,65 @@
+definition {
+
+	macro addCases {
+		Click(locator1 = "Testray#ADD_PROJECT");
+
+		Type(
+			locator1 = "TestrayManage#NEW_OPTION_NAME",
+			value1 = "${caseName}");
+
+		Select(
+			locator1 = "TestrayCases#CASE_PRIORITY",
+			value1 = "${Priority}");
+
+		Select(
+			locator1 = "TestrayCases#CASE_TYPE",
+			value1 = "${CaseTypeId}");
+
+		Select(
+			locator1 = "TestrayCases#MAIN_COMPONENT",
+			value1 = "${ComponentId}");
+
+		Type(
+			locator1 = "TestrayCases#ESTIMATED_DURATION",
+			value1 = "${Duration}");
+
+		Type(
+			locator1 = "TestrayCases#FIELD_DESCRIPTION",
+			value1 = "${Description}");
+
+		Type(
+			locator1 = "TestrayCases#FIELD_STEPS",
+			value1 = "${Steps}");
+
+		Click(locator1 = "TestrayManage#SAVE_BUTTON");
+
+		Refresh();
```

Comment:
@joelrocha68 why do we need a refresh here?

Reply:
It needs a refresh because when creating a Cases it does not appear on the page, but refreshing it appears.